sdk/src/wallet/migration/migrate_0.rs

```diff
@@ -64,14 +66,27 @@ impl Migration for Migrate {
                 }
             }
         }
+
+        if let Some(mut wallet) = storage.get::<serde_json::Value>(WALLET_INDEXATION_KEY).await? {
+            ConvertHrp::check(
+                wallet
+                    .get_mut("client_options")
```

Comment:
Do we really have a mix of snake_case and camelCase in the storage? Should we migrate it?

Reply:
We do and we can if we want to. Different PR tho I think.

Reply:
https://github.com/iotaledger/iota-sdk/issues/482